ubuntu.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+# list of package installed through apt-get 
+# (required to run this scripts and/or as dependancies for PyNE and its depedancies)
+apt_package_list="software-properties-common \
+                  python3-pip \
+                  wget \
+                  build-essential \
+                  git \
+                  cmake \
+                  gfortran \
+                  libblas-dev \
+                  liblapack-dev \
+                  libeigen3-dev \
+                  libhdf5-dev \
+                  hdf5-tools"
+
+# list of python package required for PyNE and its depedencies (installed using pip3 python package manager)
+pip_package_list="numpy \
+                  scipy \
+                  cython \
+                  nose \
+                  tables \
+                  matplotlib \
+                  jinja2 \
+                  setuptools \
+                  future"
+
+# hdf5 std directory
+hdf5_libdir=/usr/lib/x86_64-linux-gnu/hdf5/serial
```

Comment:
Is this true on all ubuntu versions?

Reply:
this worked on my test on ubuntu 16.04 18.04 and 20.04

Reply:
but I did not verified it

Reply:
I didn't have issues on the 20.04 dockerfile so I believe this is still the correct path